ballerina/modules/files/tests/test.bal

```diff
@@ -216,7 +217,12 @@ function testGetFileMetadata() returns error? {
 }
 function testLargeFileUpload() returns error? {
     byte[] actualContent = check io:fileReadBytes(path = resourcesPath + testLargeFileName);
+    byte[] hashKey = "test-key".toBytes();
+    byte[] md5Hash = check crypto:hmacMd5(actualContent, hashKey);
     check fileClient->directUploadFileAsByteArray(testFileShareName, actualContent, "largeFile.txt");
+    byte[] downloadedContent = check fileClient->getFileAsByteArray(testFileShareName, "largeFile.txt");
+    byte[] downloadedMd5Hash = check crypto:hmacMd5(downloadedContent, hashKey);
+    test:assertTrue(md5Hash == downloadedMd5Hash, "MD5 hash mismatch");
```

Comment:
Why can't we directly compare `actualContent` and `downloadedContent `?

Reply:
Fixed with 81943d21eb1869b24a70c9f1649d84da111b8ca4